project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   val sentryRavenLogback = "com.getsentry.raven" % "raven-logback" % "7.2.3"
   val scalaUri = "com.netaporter" %% "scala-uri" % "0.4.6"
   val memsubCommonPlayAuth = "com.gu" %% "memsub-common-play-auth" % "0.7"
-  val membershipCommon = "com.gu" %% "membership-common" % "0.317"
+  val membershipCommon = "com.gu" %% "membership-common" % "0.320-SNAPSHOT"
```

Comment:
Am I right in thinking that the changes to common will now be published as `0.319`? Except if @pvighi merges and releases [his changes](https://github.com/guardian/membership-common/pull/370) first, and then it should be `0.320`?